src/steam_network/mvc_classes.py

```diff
@@ -1,46 +1,51 @@
 """ mvc_classes.py
 
-A collection of classes that the steam network model-view-controller will use to share data between them. while the original code tended to group classes where they were most used, splitting them to a dedicated folder is a better way to avoid circular references. 
+A collection of classes that the steam network model-view-controller will use to share data between them. while the original code tended to group classes where they were most used, splitting them to a dedicated folder is a better way to avoid circular references.
 
 """
 from __future__ import annotations
 
-from abc import ABC, abstractmethod
 from enum import Enum, IntEnum
-from typing import NamedTuple, Dict, Optional, List
+from typing import List, NamedTuple
+
 from rsa import PublicKey
 
-from .protocol.messages.steammessages_auth import CAuthentication_AllowedConfirmation, EAuthSessionGuardType
-#implement error enum for use with website. 
+from .protocol.messages.steammessages_auth import (
+    CAuthentication_AllowedConfirmation, EAuthSessionGuardType)
+
+
+# implement error enum for use with website.
 
-#a collection of error codes the auth flow can produce that the view knows how to handle. this typically means sending the right query string parameter to the webpage or things like that.
-#also has a generic unknown value, which just tells the view "i dunno, give them this message i guess"
+# a collection of error codes the auth flow can produce that the view knows how to handle. this typically means sending the right query string parameter to the webpage or things like that.
+# also has a generic unknown value, which just tells the view "i dunno, give them this message i guess"
 class AuthErrorCode(IntEnum):
```

Comment:
IMO enums should have their values lined up for readability, but considering we don't do it anywhere else in the code (and most of these i don't really care about the underlying value now that i have it implemented), this is fine. 

An aside, does python have default values for IntEnums so i can just leave them implicit? (a la C or C# enums.) Or should i keep the values in anyway for ease of use?

Reply:
I don't know Enum in Python well enough to answer that question